Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp

```diff
@@ -1461,6 +1464,19 @@ bool msblenContext::sendObjects(MeshSyncClient::ObjectScope scope, bool dirty_al
 
     m_property_manager.clearReceivedData();
 
+    // If a python callback was requested, that may change the scene so run it and don't export until next update:
+    if (m_server_requested_python_callback) {
```

Comment:
Could the python_callback be renamed to client_callback? The comment applies for all the naming in this PR. I think the purpose if this functionality it to allow the server to ask the client for invoking custom callbacks, if it's in python or not sounds irrelevant

Reply:
Renamed it now